### PR TITLE
Add release notes and highlights for 2.7.0

### PR DESCRIPTION
**Pull request by thbkrkr**
Please fully review docs/release-notes/2.7.0.asciidoc which I inadvertently added early in #6533 :man_facepalming:.

### CODE REVIEW

File: docs/release-notes/highlights-2.7.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[[release-highlights-2.7.0]]
+== 2.7.0 release highlights
+
+[float]
+[id="{p}-270-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.7.0 of {n}. Check <<release-notes-2.7.0>> for the full list of changes.
```
Comment (pebrc) [inline docs/release-notes/highlights-2.7.0.asciidoc:8]: Should we have at least one meaning full sentence (or leave out release hightligths altogether). Something along the lines of this is a maintenance release see the release notes for full list of changes.

**Review by pebrc — APPROVED** [review]

File: docs/release-notes/highlights-2.7.0.asciidoc (hunk 1 above)
Comment (thbkrkr) [reply docs/release-notes/highlights-2.7.0.asciidoc:8]: Yes, I highlighted 2 changes in [20944b5](https://github.com/elastic/cloud-on-k8s/pull/6566/commits/20944b504ec84aa5b75a7170364d4234723f71a1).

**Comment by thbkrkr** [comment]
buildkite test this

File: docs/release-notes/highlights-2.7.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+=== ECK Beats to Elastic Cloud
+
+With the improvements made in this and multiple previous releases, ECK-managed Beats can connect to non-ECK-managed Elastisearch,
+such as a monitoring Elasticsearch cluster hosted on Elastic Cloud.
+
+[float]
+[id="{p}-270-eck-es-helm-chart"]
+=== ECK Elasticsearch Helm chart
+
+The eck-elasticsearch Helm chart is now iso functional with what is possible when a raw k8s manifest is used and
```
Comment (pebrc) [inline docs/release-notes/highlights-2.7.0.asciidoc:22]:
```suggestion
The eck-elasticsearch Helm chart exposes now all functionality that is available through the ECK Elasticsearch manifest and
```

**Review by pebrc — APPROVED** [review]

**Pull request merged**